opm/simulators/timestepping/AdaptiveTimeSteppingEbos.hpp

```diff
@@ -288,7 +297,8 @@ namespace Opm {
             , growthFactor_(tuning.TFDIFF)
             , maxGrowth_(tuning.TSFMAX)
             , maxTimeStep_(tuning.TSMAXZ) // 365.25
-            , minTimeStep_(unitSystem.to_si(UnitSystem::measure::time, EWOMS_GET_PARAM(TypeTag, double, SolverMinTimeStep))) // 1e-12;
+            , minTimeStep_(tuning.TSFMIN) // 0.1;
+            , ignoreConvergenceFailure_(true)
```

Comment:
Looks strange. Shouldn't you query the parameter or use the default (`false`)

Reply:
The logic is as following. 
If --enable-tuning=true
we honor the TUNING keyword. 
where TSFMIN is the minimum step size for which all time steps are accepted regardless of convergence. 
That is at least my understanding. @OPMUSER Isn't that right?

Reply:
I see. Hence this will get set by the Tuning keyword somewhere? Maybe we should add a comment somewhere (for the case others are on my dumbness/smartness level)?

Reply:
Sorry for the late reply, been "enjoying: GitHub PRs :-(

@totto82 Yes that is correct, you get a message saying something like, None Convergence Time Step Accepted as Too Small to Chop`